src/platform/forms-system/src/js/review/StringField.jsx

```diff
@@ -13,18 +14,69 @@ import {
  * in the config
  */
 export default function StringField(props) {
-  const { registry, schema, uiSchema, formData } = props;
-  const uiOptions = getUiOptions(uiSchema);
+  const {
+    registry = getDefaultRegistry(),
+    schema,
+    uiSchema,
+    formData,
+    children,
+    formContext,
+    id,
+  } = props;
+  const realUiSchema = checkForNestedUISchema(uiSchema);
+  const uiOptions = getUiOptions(realUiSchema);
   const labels = uiOptions.labels || {};
   const enumOptions = Array.isArray(schema.enum) && optionsList(schema);
+  const isRoot = id === 'root';
+  let Widget = _.get('ui:reviewWidget', realUiSchema);
 
-  let Widget = _.get('ui:reviewWidget', uiSchema);
   if (!Widget) {
-    const defaultWidget = schema.format || (enumOptions ? 'select' : 'text');
-    Widget = getWidget(
-      schema,
-      uiOptions.widget || defaultWidget,
-      registry.widgets,
+    if (schema.type !== 'object' && schema.type !== 'array') {
+      const defaultWidget = schema.format || (enumOptions ? 'select' : 'text');
+      Widget = getWidget(schema, defaultWidget, registry.widgets);
+    } else {
+      return children;
+    }
+  }
+
+  if (
+    (isRoot && schema.type === 'object') ||
+    (isRoot && schema.type === 'array')
+  ) {
+    let title = formContext.pageTitle;
+    if (!formContext.hideTitle && typeof title === 'function') {
+      title = title(formData, formContext);
+    }
+    const editLabel =
+      _.get('ui:options.ariaLabelForEditButtonOnReview', realUiSchema) ||
```

Comment:
Like above, is there a reason that we're using lodash here instead of using a standard property accessor?

Reply:
I copied the implementation from other places that have it such as `ObjectField` and `ArrayField`. But I am happy to modernize this!